src/pages/elements/app-icons.mdx

```diff
@@ -9,3 +9,1172 @@ They allow users to identify apps at a glance while serving as a unique
 expression of our brand.
 
 </PageDescription>
+
+<AnchorLinks>
+  <AnchorLink>Overview</AnchorLink>
+  <AnchorLink>Stroke icons</AnchorLink>
+  <AnchorLink>Fill icons</AnchorLink>
+  <AnchorLink>IBM Plex icons</AnchorLink>
+  <AnchorLink>Hero brands and 3rd parties</AnchorLink>
```

Comment:
Noticed that this AnchorLink doesn't link to that section of the page.

Reply:
I updated the heading below, should work now 👍